resources/views/dashboard.blade.php

```diff
@@ -5,115 +5,66 @@
 
     <div>
         <div class="row">
+            <div class="col-md-6 col-xl-6 mb-2">
+                @include('lead.partials._list', [
+                    'leads' => $leads, 
+                    'url' => url('/lead'),
+                    'title' => __('Leads')
+                ])
+                <!-- Last Leads -->
+
+                @include('lead.partials._list', [
```

Comment:
I like the idea but maybe we can rename as something more generic if we will use the same for lead an customer.